### ISSUE TITLE
git-lfs-2.9.0-1 FTBFS

### ISSUE THREAD

**Issue by dmacks** [fink/fink-distributions#714]
On my 10.13 box:
```
#!/bin/sh -ev
	export GOPATH=/sw/build.build/git-lfs-2.9.0-1/git-lfs-2.9.0
	
	rm $GOPATH/go.mod
	mkdir -p src/github.com/git-lfs
	ln -s /sw/build.build/git-lfs-2.9.0-1/git-lfs-2.9.0 src/github.com/git-lfs/git-lfs
	
	pushd  src/github.com/git-lfs/git-lfs
/sw/build.build/git-lfs-2.9.0-1/git-lfs-2.9.0/src/github.com/git-lfs/git-lfs /sw/build.build/git-lfs-2.9.0-1/git-lfs-2.9.0
	make
fatal: not a git repository (or any of the parent directories): .git
[that 'fatal:...' repeated many times]
 \

[repeated many times]
make: *** [Makefile:438: lint] Error 1
```

**Comment by nieder** [comment]
GNU make issue. See #511

**Comment by dmacks** [comment]
Fixed by #511
